dpnp/backend/extensions/blas/CMakeLists.txt

```
@@ -84,7 +84,7 @@ if (DPNP_GENERATE_COVERAGE)
 endif()
 
 if(_use_onemkl_interfaces)
-    target_link_libraries(${python_module_name} PUBLIC onemkl)
+    target_link_libraries(${python_module_name} PUBLIC ${USED_MATH_LIBRARY})
```

following oneMath [guide](https://github.com/uxlfoundation/oneMath/blob/af66e5ec23913030e3dbb577a176377a07070e14/docs/using_onemath_with_cmake.rst) it should be `PRIVATE`:
```suggestion
    target_link_libraries(${python_module_name} PRIVATE ${USED_MATH_LIBRARY})
```